cypress/cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+    "baseUrl": "http://localhost:3001"
+}
```

Comment:
put an empty line at the end of file

Reply:
those are eslint rules :), I will disable the rule